src/components/container/FeaturedJobs/FeaturedJobs.scss

```diff
@@ -0,0 +1,30 @@
+.FeaturedJobs {
+    display: flex;
+    flex-direction: column;
+    gap: 59px;
+
+    max-width: var(--section-max-width);
+    padding: 96px 0;
```

Comment:
I don't think we have such a global variable

Reply:
You're right! I thought we might use one in the future, but I removed it for now